web/cats/static/cats/cat_profile_style.css

```diff
@@ -1,17 +1,54 @@
+*{
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+/*Style for a header*/
+header {
+  position: fixed;
+  top:0; left:0; right:0;
+  text-align: center;
+  justify-content: space-around;
+  display: flex;
+  background-color: white;
+}
+
+header .navbar a {
+  font-size: 1.4rem;
+  color: black;
+  background-color: beige;
+  display: inline-block;
+  margin-top: 10px;
+}
+
+header .navbar a:hover {
+  color: red;
+
+}
+
+header .logo {
+  position: fixed;
```

Comment:
Are you sure you need to set `position: fixed` inside an element which already has `position: fixed`? Do you expect that parent element (`<header>`) to be scrollable?